source/quickstart.txt

```diff
@@ -1,12 +1,235 @@
+         mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,clusterTwo02.fancyCorp.com:20020,clusterTwo03.fancyCorp.com:20020
+
+      For more details, see :ref:`c2c-connecting`.
+
+Initialize ``monogsync``
+------------------------
+
+``monogsync`` must create an initial connection to the source and
+destination clusters before it can start to sync data. To create the
+initial connection, issue the following command on a single line (the
+command is reformated here for clarity):
+
+.. code-block:: shell
+
+   ./bin/mongosync --cluster0 mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,
+                              clusterOne02.fancyCorp.com:20020,
+                              clusterOne03.fancyCorp.com:20020
+                   --cluster1 mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,
+                              clusterTwo02.fancyCorp.com:20020,
+                              clusterTwo03.fancyCorp.com:20020
+
+Initialization Notes
+~~~~~~~~~~~~~~~~~~~~
+
+- When ``mongosync`` first connects to the source and destination
+  clusters it is in the :ref:`IDLE <c2c-states-descriptions>` state.
+- ``mongosync`` does not synchronize data until it receives the
+  :ref:`start <c2c-api-start>` command. 
+- Designate the source and destination clusters with the :ref:`start
+  <c2c-api-start>` command. "cluster0" and "cluster1" are just labels,
+  either cluster can be ``cluster0`` or ``cluster1``.
+
+Synchronize Data Between Clusters
+---------------------------------
+
+The command interface for ``mongosync`` is an HTTP server that
+publishes a REST API. To control ``mongosync``, use the API endpoints.
+
+The :ref:`start <c2c-api-start>` command initiates data synchronization.
+To start synchronizing, use ``curl`` or a similar program to issue the
+:ref:`start <c2c-api-start>` command:
+
+.. literalinclude:: /includes/api/requests/start.sh
+   :language: shell
+
+Once started, ``mongosync`` will synchronize existing data on the
+source cluster with the destination cluster. After the initial sync has
+completed, new writes to the source cluster will be synchronized with
+the destination cluster.
+
+The API documentation provides details on using the following endpoints:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Endpoint
+     - Description
+ 
+   * - :ref:`c2c-api-commit`
+     - Commits the synchronization operation to the destination
+       cluster.
+   * - :ref:`c2c-api-pause`
+     - Pauses the current synchronization operation.
+   * - :ref:`c2c-api-progress`
+     - Returns the status of the synchronization process.
+   * - :ref:`c2c-api-resume`
+     - Resumes a paused synchronization session based on data stored on
+       the destination cluster.
+   * - :ref:`c2c-api-reverse`
+     - Reverses the direction of a committed sync operation.
+   * - :ref:`c2c-api-start`
+     - Starts the synchronization between a source and destination
+       cluster.
+
+Synchronization Notes
```

Comment:
I think it might be good to mention that users may need to increase the file descriptor ulimits when running mongosync (particularly for macos which has low defaults). We could link to https://www.mongodb.com/docs/v5.3/reference/ulimit/

I don't know if it's best here or in some other section, but I think people testing on macos are likely to run into this problem so it's worth mentioning.

Reply:
Done